### ISSUE TITLE
How can I stop org-ref-export from capitilizing titles?

### ISSUE THREAD

**Issue by dangom** [jkitchin/org-ref#1025]
Currently, if I export an org file to latex using org-ref-export, titles in the .bib file will be capitalized on export.

Take for example:

```
@article{glover1999deconvolution,
  author =       {Gary H. Glover},
  title =        {Deconvolution of Impulse Response in Event-Related BOLD fMRI},
  journal =      {NeuroImage},
  volume =       9,
  number =       4,
  pages =        {416-429},
  year =         1999,
  doi =          {10.1006/nimg.1998.0419},
  url =          {http://dx.doi.org/10.1006/nimg.1998.0419},
}
```

lands in the tex file as:

```
\hypertarget{citeproc_bib_item_1}{Glover, Gary H. 1999. “Deconvolution of Impulse Response in Event-Related Bold Fmri.” \textit{Neuroimage} 9 (4): 416–29. doi:\href{https://doi.org/10.1006/nimg.1998.0419}{10.1006/nimg.1998.0419}.}
```

Note how BOLD fMRI was converted to Bold Fmri. The capitalization doesn't make sense for acronyms.

I don't remember ever having this issue in V2, so I believe there is something new that we users should configure?

**Comment by jkitchin** [comment]
you probably should not use org-ref-export for LaTeX. This would go through citeproc, and I guess that is not necessary. For latex, the usual org export should work fine, e.g. C-c C-e lo.

**Comment by dangom** [comment]
The problem is that I ultimately would like to convert my org to word. The route org-ref-export-to-latex, and then pandoc that latex was the best I could find so far. The only issue here is the wrong capitalization of the bibliography, everything else is perfect.

Org-ref-export-to-word also goes through citeproc, but for some reason the document title is not showing up on my exported docx, hence the latex path.

Org direct export to latex doesn't go through citeproc, but pandoc doesn't know how to handle the bibliography at all then.

```
\bibliographystyle{unsrtnat}
\bibliography{.biblio/references}
```

Neither work:
```sh
pandoc  --csl styles/elsevier-harvard.csl   --bibliography biblio/references.bib  -s manuscript.tex -o manuscript.docx
 pandoc -F pandoc-crossref --bibliography biblio/references.bib  -s manuscript.tex  -o manuscript.docx
```


I don't know if you ever had issues with pandoc and bibliography before. I'm using elsarticle.
Any ideas or suggestions would be greatly appreciated, but feel free to close the issue if you think it's unrelated to org-ref.

Thank you.

**Comment by jkitchin** [comment]
I guess the capitalization is due to the csl file. org-ref doesnt' touch that, it takes what the citeproc library returns. Maybe you need a different csl style?


I get this output:

<img width="868" alt="image" src="https://user-images.githubusercontent.com/1610070/184161918-18b9472d-6b06-4aeb-8c44-2a5d99d0e42b.png">


from 

<img width="1490" alt="image" src="https://user-images.githubusercontent.com/1610070/184161999-a3c70939-506a-4f1d-ac72-7bdbb82e770a.png">


with C-x C-e rw

**Comment by dangom** [comment]
Uhmm.. it is "capitalizing" in the sense that it is only keeping the first letter of acronyms capitalized, right? In the example above we would expect the journal name to be "ACS Catalysis" and not "Acs catalysis". Or is Acs in your bibfile?

The same happens here:

<img width="731" alt="image" src="https://user-images.githubusercontent.com/7881842/184214377-e2b257c3-2576-4349-8744-040a9c98d3b3.png">

Where bold fmri should be BOLD fMRI, and Neuroimage should be NeuroImage.

After some investigation, I found this here:
https://github.com/jkitchin/org-ref/blob/master/org-ref-export.el#L333

But this doesn't really seem to be causing the issue, as I get the same results if I remove it.

**Comment by jkitchin** [comment]
Something seems off. This code does the right thing I think

```
#+BEGIN_SRC emacs-lisp
(defun render (csl-style data)
  (let* ((proc (citeproc-create csl-style
				(citeproc-itemgetter-from-bibtex "../org-ref.bib")
				(citeproc-locale-getter-from-dir "../citeproc/csl-locales/")
				"en-US"))
	 (cites (list (citeproc-citation-create :cites data
						:mode 'nil
						:capitalize-first nil
						:suppress-affixes nil)))
	 (rendered-citations (progn (citeproc-append-citations cites proc)
				    (citeproc-render-citations proc 'plain nil)))
	 (bib (citeproc-render-bib proc 'org))
	 )
    (car bib)))

;; https://raw.githubusercontent.com/citation-style-language/styles/master/elsevier-harvard.csl
(render "elsevier-harvard.csl" '(((id . "kitchin-2015-examp")
				  (prefix . "See ")
				  (suffix . "")
				  (locator . "2")
				  (label . "page")
				  (suppress-author))))
#+END_SRC

#+RESULTS:
: <<citeproc_bib_item_1>>Kitchin, J.R., 2015. Examples of Effective Data Sharing in Scientific Publishing. ACS Catalysis 5, 3894–3899. https://doi.org/10.1021/acscatal.5b00538
```

I think org-ref-export.el does the same thing, but the capitalization is different. I have not found where the difference is yet.

**Comment by jkitchin** [comment]
I have identified the issue, although, I am not sure what the right solution is. What you see happening is caused by `(citeproc-hash-itemgetter-from-any (org-ref-find-bibliography))`.  if you look at the source of org-ref-export.el, above that line you see:

    ;; (citeproc-itemgetter-from-bibtex (org-ref-find-bibliography))

so obviously (to me) I once used that, and when you do the capitalization is correct. with `citeproc-hash-itemgetter-from-any`, it is not correct. 

This was changed here https://github.com/jkitchin/org-ref/commit/4a465be9f8a4093a4afa22abc77a2012624f5e90.

I feel like it is an issue in citeproc (@andras-simony) Looking at the example above, with `citeproc-itemgetter-from-bibtex`, the bibliography is rendered as: 

    <<citeproc_bib_item_1>>Kitchin, J.R., 2015. Examples of Effective Data Sharing in Scientific Publishing. ACS Catalysis 5, 3894–3899. https://doi.org/10.1021/acscatal.5b00538

but with  `citeproc-hash-itemgetter-from-any` it is rendered as:

     <<citeproc_bib_item_1>>Kitchin, J.R., 2015. Examples of effective data sharing in scientific publishing. Acs catalysis 5, 3894–3899. https://doi.org/10.1021/acscatal.5b00538

Note the difference in capitalization.

**Comment by andras-simonyi** [comment]
Yes, since CSL styles assume that title fields are in title-case, citeproc-el with the recommended itemgetter now capitalizes bib(la)tex title fields during the bib(la)tex->CSL conversion by default (see [issue #71](https://github.com/andras-simonyi/citeproc-el/issues/71) for details). The algorithm is quite dumb so acronyms etc have to protected by curly brackets, e.g., 
    
    title =        {Deconvolution of Impulse Response in Event-Related {BOLD fMRI}}

the converter function has a switch for turning this behaviour off but currently there is no "UI" (e.g., a variable) to access it because nobody requested it.

**Comment by dangom** [comment]
Very interesting. Thank you for sharing your investigation.
I would assume that the correct behaviour is to copy the title verbatim from what is in the bibfile unless otherwise specified, right? 
Would there be an easy way to do that?

Protecting acronyms with curly brackets seems reasonable, but it's extra manual work for the end-user. My (potentially suboptimal) workflow for reference management consists of opening pdfs on emacs and calling org-ref-pdf-to-bibtex on them. Although I could add brackets to each entry, this seems like extra room for error.

**Comment by andras-simonyi** [comment]
> Very interesting. Thank you for sharing your investigation. I would assume that the correct behaviour is to copy the title verbatim from what is in the bibfile unless otherwise specified, right? 

If you mean that the formatted title in the generated bibliography should always contain the content of the `title` field verbatim then this is definitely not the case, a lot of CSL styles require case transformations when the input is in sentence-case as assumed.

If you simply want to turn off the default sentence casing conversion between bib(la)tex and CSL then I'm open to adding a customizable option (variable) for that if you open an issue in ˙citeproc-el` repository.

**Comment by dangom** [comment]
just a note that the conversion for titles also happens when importing bibtex entries using doi-utils:

```
;; Test
(doi-utils-insert-bibtex-entry-from-doi "10.1006/nimg.1998.0419")
```

**Comment by jkitchin** [comment]
When you insert bibtex entries, there isn't a mechanism for protecting words like BOLD fMRI1, you have to do those manually by wrapping them in {}.
